### ISSUE TITLE
Add build instructions or binaries

### ISSUE THREAD

**Issue by HolgerJeromin** [openstreetview/upload-GUI#4]
I am sure not many users are able to build your tool with this few information in the Readme.

- Windows users probably want a binary.
- Medium advanced Linux users need step by step instructions 

What the hack is a ".pro" file and what to do with it? make, cmake, configure are known to me, but I am completely lost..

**Comment by ghost** [comment]
@HolgerJeromin I figured out the build process and packaged this software for the Arch User Repositories. The package contains a build script (PKGBUILD) that shows how you can build this application.

https://aur.archlinux.org/packages/openstreetcam-gui/

**Comment by ghost** [comment]
You can view the PKGBUILD here: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=openstreetcam-gui

If you're not packaging but installing manually, this should work (create directories when necessary):

* clone upload-GUI and cd into the directory (in these instructions I'll call that directory `<repo>`)
* apply [the patch](https://aur.archlinux.org/cgit/aur.git/plain/openstreetcam-gui-r15.9ac859e.patch?h=openstreetcam-gui) I wrote
* cd into `OSV_Qt_upload`
* run `qmake-qt5`
* run `make`
* create a directory `/usr/local/lib/openstreetcam-gui/` and copy these things into it (do not create subdirectories, just copy all the files):
    * `<repo>/OSV_Qt_upload/KQOAuth/libKQOAuth.so*`
    * `<repo>/OSV_Qt_upload/HTTPRequest/libHTTPRequest.so*`
    * `<repo>/OSV_Qt_upload/UploadComponent/UploadComponent`
* install the [shell script](https://aur.archlinux.org/cgit/aur.git/plain/openstreetcam-gui.sh?h=openstreetcam-gui) I wrote as `/usr/local/bin/openstreetcam-gui`, but change `/usr/lib` in the file to `/usr/local/lib` (two occurrences)
* make sure the shell script has its executable flag set, by executing `chmod +x /usr/local/bin/openstreetcam-gui`

At this point you should be able to launch the program by executing `/usr/local/bin/openstreetcam-gui`.

To install an entry in your launcher:

* install the [logo](https://aur.archlinux.org/cgit/aur.git/plain/openstreetcam-logo.png?h=openstreetcam-gui) as `/usr/local/share/pixmaps/openstreetcam.png`
* install the [.desktop file](https://aur.archlinux.org/cgit/aur.git/plain/openstreetcam-gui.desktop?h=openstreetcam-gui) I wrote as `/usr/local/share/applications/openstreetcam-gui`, but change `/usr/bin` in the file to `/usr/local/bin` (one occurrence)

**EDIT 15:35 UTC**: corrected instructions

**Comment by ghost** [comment]
Update: these instructions don't work any more on up-to-date systems. The software just cannot be compiled any more.

**Comment by alexandrui-telenav** [comment]
True, unfortunately. The developer continued working on this and improved the tool but cannot do a new release as currently he's assigned on a different project. Archiving the repo for now.
